lib/ddtrace/metrics.rb

```diff
@@ -279,7 +280,7 @@ def ignored_statsd_warning
       IGNORED_STATSD_ONLY_ONCE.run do
         Datadog.logger.warn(
           'Ignoring user-supplied statsd instance as currently-installed version of dogstastd-ruby is incompatible. ' \
-          "To fix this, ensure that you have `gem 'dogstatsd-ruby', '~> 4.0'` on your Gemfile or gems.rb file."
+          "To fix this, ensure that you have `gem 'dogstatsd-ruby', '>= 5.2'` on your Gemfile or gems.rb file."
```

Comment:
Should this be `gem 'dogstatsd-ruby', '~> 5.2'` (also on the `GettingStarted.md`)?

Reply:
phew, yes thanks. (sanity check: it will still upgrade to 5.3, 5.4, ..., ?)

Reply:
Precisely -- you can see in the docs that apparently nobody calls this the twiddle-wakka: https://guides.rubygems.org/patterns/#pessimistic-version-constraint (lol and I don't either)